tests/store/artifact/test_local_artifact_repo.py

```diff
@@ -41,24 +41,34 @@ def test_log_artifacts(local_artifact_repo, local_artifact_root):
 
     artifacts_list = local_artifact_repo.list_artifacts()
     assert len(artifacts_list) == 1
-    assert artifacts_list[0].path == artifact_rel_path
+    assert (artifacts_list[0].path == artifact_rel_path)
 
     artifact_dst_path = os.path.join(local_artifact_root, artifact_rel_path)
     assert os.path.exists(artifact_dst_path)
     assert artifact_dst_path != artifact_src_path
     assert open(artifact_dst_path).read() == artifact_text
 
 
-def test_download_artifacts(local_artifact_repo):
+@pytest.mark.parametrize("dst_path", [None, "dest"])
+def test_download_artifacts(local_artifact_repo, dst_path):
     artifact_rel_path = "test.txt"
     artifact_text = "hello world!"
+    empty_dir_path = "empty_dir"
     with TempDir(chdr=True) as local_dir:
+        if dst_path:
+            os.mkdir(dst_path)
```

Comment:
Nit: Can we call `os.mkdir(local_dir.path(dst_path))` instead so that the directory is cleaned up later?